lib/open_weather/endpoints/stations.rb

```diff
@@ -10,6 +10,19 @@ def register_station(options = {})
       def list_stations
         get('stations').map { |data| OpenWeather::Models::Station.new(data) }
       end
+
+      def get_station(id)
+        OpenWeather::Models::Station.new(get("stations/#{id}"))
+      end
+
+      def update_station(id, options = {})
+        OpenWeather::Models::Station.new(put("stations/#{id}", options))
+      end
+
+      def delete_station(id)
+        delete("stations/#{id}")
+        nil
```

Comment:
@dblock what would you suggest to return here? Currently it the `delete` method returns as empty string (`response.body`).

Reply:
I think `nil` is fine.